pkg/api/v1/types.go

```diff
@@ -1220,6 +1226,33 @@ type ScaleIOVolumeSource struct {
 	ReadOnly bool `json:"readOnly,omitempty" protobuf:"varint,10,opt,name=readOnly"`
 }
 
+// Represents a StorageOS persistent volume resource.
+type StorageOSVolumeSource struct {
+	// VolumeName is the human-readable name of the StorageOS volume.  Volume
+	// names are only unique within a namespace.
+	VolumeName string `json:"volumeName,omitempty" protobuf:"bytes,1,opt,name=volumeName"`
+	// Namespace specifies the scope of the volume name within StorageOS.  If no
+	// namespace is specified, the namespace of the pod will be used.  Set to
+	// "default" if you are not using namespaces within StorageOS.
+	// +optional
+	Namespace string `json:"namespace,omitempty" protobuf:"bytes,2,opt,name=namespace"`
```

Comment:
Is there something else we can call this? I'm concerned it will be confused for a kubernetes namespace and used incorrectly